shared/types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sjcrh/proteinpaint-types",
-  "version": "2.124.0",
+  "version": "2.125.0",
```

Comment:
Did this get bumped automatically or manually? I think the versions in the shared/types, client front workspaces should not be bumped, as there are no code changes that occurred in these workspaces.